apps/openmw/mwmechanics/character.cpp

```diff
@@ -1542,9 +1542,10 @@ void CharacterController::update(float duration)
                 mAnimation->disable(mAnimQueue.front().mGroup);
                 mAnimQueue.pop_front();
 
+                bool loopfallback = (mAnimQueue.front().mGroup.compare(0,4,"idle") == 0);
```

Comment:
I had wondered about that, but it seemed that it was already being converted to lower-case. When I typed "playgroup Idle2" and checked against "Idle," I got a false.